rpc/transport.go

```diff
@@ -67,7 +71,8 @@ type transport struct {
 	protocols  *protocolHandler
 	calls      *callContainer
 	log        LogInterface
-	startCh    chan struct{}
+	closeOnce  *sync.Once
+	startOnce  *sync.Once
```

Comment:
Why are these pointers instead of a `sync.Once()`?

Reply:
Oh I was under the impression we've been using `transport` instead of `*transport` but it turns out I was wrong. Will fix thanks!